roles/download/defaults/main.yml

```diff
@@ -1001,9 +1001,9 @@ multus_image_tag: "{{ multus_version }}"
 kube_vip_image_repo: "{{ github_image_repo }}/kube-vip/kube-vip"
 kube_vip_image_tag: v0.5.5
 nginx_image_repo: "{{ docker_image_repo }}/library/nginx"
-nginx_image_tag: 1.23.0-alpine
+nginx_image_tag: 1.23.2-alpine
```

Comment:
The job is failed:
```
TASK [download : download_container | Download image if required] **************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/roles/download/tasks/download_container.yml:55
Monday 21 November 2022  03:39:14 +0000 (0:00:00.094)       0:04:50.169 ******* 
FAILED - RETRYING: download_container | Download image if required (4 retries left).
FAILED - RETRYING: download_container | Download image if required (3 retries left).
FAILED - RETRYING: download_container | Download image if required (2 retries left).
FAILED - RETRYING: download_container | Download image if required (1 retries left).
fatal: [instance-1 -> instance-1]: FAILED! => {"attempts": 4, "changed": true, "cmd": ["/usr/local/bin/nerdctl", "-n", "k8s.io", "pull", "--quiet", "quay.io/kubespray/nginx:1.23.2-alpine"], "delta": "0:00:00.414519", "end": "2022-11-21 03:39:46.384432", "msg": "non-zero return code", "rc": 1, "start": "2022-11-21 03:39:45.969913", "stderr": "time=\"2022-11-21T03:39:46Z\" level=info msg=\"trying next host - response was http.StatusNotFound\" host=quay.io\ntime=\"2022-11-21T03:39:46Z\" level=fatal msg=\"failed to resolve reference \\\"quay.io/kubespray/nginx:1.23.2-alpine\\\": quay.io/kubespray/nginx:1.23.2-alpine: not found\"", "stderr_lines": ["time=\"2022-11-21T03:39:46Z\" level=info msg=\"trying next host - response was http.StatusNotFound\" host=quay.io", "time=\"2022-11-21T03:39:46Z\" level=fatal msg=\"failed to resolve reference \\\"quay.io/kubespray/nginx:1.23.2-alpine\\\": quay.io/kubespray/nginx:1.23.2-alpine: not found\""], "stdout": "", "stdout_lines": []}
```
The important part seems `quay.io/kubespray/nginx:1.23.2-alpine: not found`

Reply:
I think so, the image should be pushed to the quay.io. But I have no write access permission :-)

Reply:
I'll try to give it to you asap, work trip right now so I'm having a hard time following everything

Reply:
@floryut Thank you for helping us for doing that.